SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -4,28 +4,110 @@
 
 #include "Schema/UnrealObjectRef.h"
 
+DEFINE_LOG_CATEGORY(LogRPCContainer);
+
 using namespace SpatialGDK;
 
+namespace
+{
+	FString ERPCErrorToString(ERPCError Error)
+	{
+		switch (Error)
+		{
+		case ERPCError::Success:
+			return "";
```

Comment:
These literal strings should use the TEXT macro.

Reply:
Updated.